jsonnet/kube-prometheus/components/k8s-control-plane.libsonnet

```
@@ -234,6 +235,36 @@ function(params) {
     },
   },
 
+  [if (defaults + params).kubeProxy then 'podMonitorKubeProxy']: {
+    apiVersion: 'monitoring.coreos.com/v1',
+    kind: 'PodMonitor',
+    metadata: {
+      labels: {
+        'k8s-app': 'kube-proxy',
+      },
+      name: 'kube-proxy',
+      namespace: k8s._config.namespace,
+    },
+    spec: {
+      jobLabel: 'k8s-app',
+      namespaceSelector: {
+        matchNames: [
+          'kube-system',
+        ],
+      },
+      selector: {
+        matchLabels: {
+          'k8s-app': 'kube-proxy',
+        },
+      },
+      podMetricsEndpoints: [{
+        honorLabels: true,
+        targetPort: 10249,
```

I would relabel the node name onto the `instance` label like the other DaemonSets:
https://github.com/prometheus-operator/kube-prometheus/blob/a330e8634ac6bd7e628d6424f48350f3cd6218b4/jsonnet/kube-prometheus/components/node-exporter.libsonnet#L151-L159

```suggestion
        targetPort: 10249,
        relabelings: [ 
          { 
            action: 'replace', 
            regex: '(.*)', 
            replacement: '$1', 
            sourceLabels: ['__meta_kubernetes_pod_node_name'], 
            targetLabel: 'instance', 
          }, 
        ], 
```